### ISSUE TITLE
Documentation ! 

### ISSUE THREAD

**Issue by sudipt1999** [sudipt1999/EDM-Mimic#6]
Kindly improve the documentation by mentioning the project details and libraries used ! Currently only howler.js is used

**Comment by petercr** [comment]
Hey I could totally help with this issue if you want 😃 
Just let me know

**Comment by sudipt1999** [comment]
Sure! Go ahead

**Comment by sudipt1999** [comment]
@petercr  are u working on the issue ! Need any help !

**Comment by petercr** [comment]
Sorry I was a little busy with some life stuff, but I'm back 😄 
I can definitely put together a README file for the project. 
**OR** if you want I could put together a Netlify site for the project. 
That way people would not have to clone the repository in order to run the project.

Either way let me know 👍

**Comment by sudipt1999** [comment]
Feel free to do anything you wish! Contribution are always welcome!

**Comment by petercr** [comment]
So just Bootstrap, FontAwesome and Howler.js ?
That's what it looks like is being used on the project. 
So I'll go ahead and add that to the README

**Comment by sudipt1999** [comment]
Yes! Just a brief description to those things , what they are used for and links to there repo! Thank you
